docs/spec/sanitations.md

```diff
@@ -1,24 +1,247 @@
-_Author_:  <!-- TODO: Add author name --> \
-_Created_: <!-- TODO: Add date --> \
-_Updated_: <!-- TODO: Add date --> \
+_Author_: Sithija Nelusha (@snelusha)
+_Created_: 18 June 2025
+_Updated_: 18 June 2025
 _Edition_: Swan Lake
 
 # Sanitation for OpenAPI specification
 
-This document records the sanitation done on top of the official OpenAPI specification from Paypal Orders. 
-The OpenAPI specification is obtained from (TODO: Add source link).
+This document records the sanitation done on top of the official OpenAPI specification from Paypal Orders.
+The OpenAPI specification is obtained from [PayPal’s official GitHub](https://github.com/paypal/paypal-rest-api-specifications/blob/main/openapi/checkout_orders_v2.json).
 These changes are done in order to improve the overall usability, and as workarounds for some known language limitations.
 
-[//]: # (TODO: Add sanitation details)
-1. 
-2. 
-3. 
+## 1. Update server URLs
 
-## OpenAPI cli command
+**Location**: `servers`
+
+**Original**:
+
+```json
+"servers": [
+  {
+    "url": "https://api-m.sandbox.paypal.com",
+    "description": "PayPal Sandbox Environment"
+  },
+  {
+    "url": "https://api-m.paypal.com",
+    "description": "PayPal Live Environment"
+  }
+]
+```
+
+**Sanitized**:
+
+```json
+"servers": [
+  {
+    "url": "https://api-m.sandbox.paypal.com/v2/checkout",
+    "description": "PayPal Sandbox Environment"
+  },
+  {
+    "url": "https://api-m.paypal.com/v2/checkout",
+    "description": "PayPal Live Environment"
+  }
+]
+```
+
+```diff
+"servers": [
+  {
+-    "url": "https://api-m.sandbox.paypal.com",
++    "url": "https://api-m.sandbox.paypal.com/v2/checkout",
+    "description": "PayPal Sandbox Environment"
+  },
+  {
+-    "url": "https://api-m.paypal.com",
++    "url": "https://api-m.paypal.com/v2/checkout",
+    "description": "PayPal Live Environment"
+  }
```

Comment:
Let's follow the common convention for the sanitizations. Remove the `diff` here. Check similar places.

Reply:
On second thoughts, let's keep the diff as it is.